powershell/plan.ps1

```diff
@@ -0,0 +1,34 @@
+$pkg_name="powershell"
+$pkg_origin="core"
+$pkg_version="6.0.0-alpha.14"
+$pkg_license=@('MIT')
```

Comment:
There is a ps linter that would be good to add soon. Basically the `"` vs `'` in powershell was inspired by rupy and so the same rules apply. I just changed to `"` for consistency.